interface/reports/dictation_review.php

```diff
@@ -0,0 +1,316 @@
+<?php
+/**
+ * Review Dictations
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Core\Header;
+
+require_once("../globals.php");
+require_once("$srcdir/patient.inc");
+require_once("$srcdir/formatting.inc.php");
+
+$from_date = fixDate($_POST['form_from_date'], date('Y-m-d'));
+$to_date   = fixDate($_POST['form_to_date'], date('Y-m-d'));
```

Comment:
Don't use fixDate since this will not support dd/mm/yyyy format. Best to use isset command here:
``````
$from_date = isset($_POST['form_from_date']) ? $_POST['form_from_date'] : date('Y-m-d');
$to_date = isset($_POST['form_to_date']) ? $_POST['form_to_date'] : date('Y-m-d');
``````
And if you want this form to support full internationalization of dates, then there is a way to do this, but that will be a little more complicated (let me know and I'd be glad to guide you through that).

Reply:
Yes, I would like you to guide me through that. On the next review.